src/components/NavBar /NavBar.jsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Modal,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { BiBookBookmark as ShoppingCartIcon } from "react-icons/bi";
+import SectionSeperator from "./SectionSeperator";
+import ShoppingCartItem from "./ShoppingCartItem";
+
+const NavBar = () => {
+  const [sidebar, setSidebar] = useState(false);
+
+  const toggleSidebar = () => {
+    setSidebar((prevState) => !prevState);
+  };
+
+  return (
+    <div>
+      <nav>
+        <div style={NavBarContainer}>
+          <div style={NavBarElement}>Digital Library</div>
+          <div style={NavBarElement}>Project Plan Builder</div>
+          <div style={NavBarElementCart}>
+            <button onClick={toggleSidebar}>
+              <ShoppingCartIcon></ShoppingCartIcon>
```

Comment:
if no need for children, then try doing `<ShoppingCartIcon />` instead. but first try to see if icon button from chakra component works